docs/DeveloperGuide.md

```diff
@@ -165,12 +165,16 @@ The `Model` component,
 <img src="images/StorageClassDiagram.png" width="600" />
 
 The `Storage` component,
-* can save address book data, task book data, sales order book data
+* Saves address book data, task book data, sales order book data
   and user preference data in json format, and read them back into corresponding objects.
-* inherits from all of `AddressBookStorage`, `TaskBookStorage`, `OrderBookStorage`,
+* The main storage class inherits from all of `AddressBookStorage`, `TaskBookStorage`, `OrderBookStorage`,
   and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
-* depends on some classes in the `Model` component (because the `Storage` component's job is to save/retrieve objects that belong to the `Model`)
-
+* Each Book Storage component (`AddressBookStorage`, `TaskBookStorage`, `OrderBookStorage`) has a `JsonSerializable` class which is in charge of converting the model's data into correct json file and retrieving the data from 
+  the json file to convert it to a model data.
+* Each `JsonSerializable` class implements its own `JsonAdapted` class which specifies methods to convert model Object
+  (i.e `Person`, `Task`, `Order`) into json object and vise versa. 
+* The `JsonSerializable` class and `JsonAdapted` class also checks the correctness of the json files format, and in the 
+  case when any of the format is wrong, it will then throw a `DataConversionException` and  `IllegalValueException`
```

Comment:
Observe that this entire chunk is under line 167

> **The `Storage` component,** 

This is why the original bullet points start with small letter verbs like **can save**, **inherits** and **depends**.

Would it be better to change line 167 to "How the `Storage` component works:" (refer to lines 116-120 and lines 133-135)?

Reply:
Thanks for the further suggestion, I have changed it!